vnext/ReactUWP/Views/ReactControl.cpp

```diff
@@ -255,5 +283,39 @@ int64_t ReactControl::GetActualWidth() const
   return static_cast<int64_t>(element.ActualWidth());
 }
 
+void ReactControl::PrepareXamlRootView(XamlView const& rootView)
+{
+  if (auto panel = rootView.try_as<winrt::Panel>())
+  {
+    // Xaml don't have blur concept.
+    // A ContentControl is created in the middle to act as a 'focus safe harbor'
+    // When a XamlView is blurred, make the ContentControl to allow tabstop, and move the pointer focus to safe harbor
+    // When the safe harbor is LosingFocus, disable tabstop on ContentControl.
+    auto children = panel.Children();
+    children.Clear();
+
+    m_focusSafeHarbor = CreateFocusSafeHarbor();
```

Comment:
Can we delay the creation of this element until its needed?

Reply:
I didn't see we have simple way to do it. Performance is not a concern since we only create once for the root control.

---
In reply to: [299606824](https://github.com/microsoft/react-native-windows/pull/2706#discussion_r299606824) [](ancestors = 299606824)

Reply:
Why dont you just do a null check on m_focusSafeHarbor in CreateFocusSafeHarbor, and call it during blur?  (Maybe rename CreateFocusSafeHarbor to EnsureFocusSafeHarbor since it would only create it if it doesn't already exist)

Reply:
Resolved. I didn't thought I only need to delay the creation of safe harbor itself.